src/component/textbox/Text.js

```diff
@@ -14,7 +14,7 @@ function Text(props) {
     return (
       <Word
         key = {index}
-        idx = {index}
+        wordLocation = {index}
```

Comment:
I would consider calling this `wordIndex`. You mention in your commit message "[it is] representing the index of the word".